tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Cacti
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+- Cacti is doing great, there is steady interest in the project as interoperability is becoming more and more important to organizations wishing to deploy distributed ledger technologies.
+- There was a significant increase in new contributors this last quarter according to LFX (75%) and the pair programming sessions get more attendance than ever.
+- The project repository was successfully moved to [its own repository](https://github.com/hyperledger-cacti) following the move from the Hyperledger Foundation to Linux Foundation's Decentralized Trust. Cacti was one of the first projects on which this experiment was carried out, and it was successful thanks to the LFDT administrators.
+- The mentorship projects are progressing nicely, with solid code contributions from the mentees, including Cactus-Weaver integration.
+- Work on keeping Cacti up-to-date with the IETF SATP interoperability protocol standard is going well.
+  * Cacti maintainers continue to be actively involved in the IETF SATP Working Group, co-authoring protocol specifications.
+- We are not doing too badly on the [OpenSSF Scorecard Metrics](https://scorecard.dev/viewer/?uri=github.com%2Fhyperledger-cacti%2Fcacti) with a present score (as of the time of this file's commit) of 6.3, but there is a lot of scope to improve.
+
+# Questions/Issues for the TAC
+
+No questions at the moment, thank you.
+
+# Releases
+
+Issued releases:
+- v2.0.0-rc.4
+- v2.0.0-rc.5
+- v2.0.0-rc.6
+- v2.0.0-rc.7
+- v2.0.0
+
+Caveat: We are still working on the publishing workflows that broke down after our latest org migration.
+v2.0.0 was affected in the sense that the artifacts are not yet published on npm due to our token not being right.
+
+Upcoming releases:
+- v2.0.1
+- v2.1.0
+
+# Overall Activity in the Past Quarter
+
+- The vast majority of the community's communication happens on the Discord channel. The mailing list is not used much nowadays.
+- The maintainers try to respond to all queries in a timely manner. On average our response time is likely around a couple of days with the tail end being a couple of weeks.
+  * Note that we don't have hard data on above from Discord, so this is just a personal estimate.
+- We are working on a number of technical initiatives: new features, improvements to the CI speed, artifact signing and improving our OpenSSF score in general.
+- The migration to the Jest test runner from tap is still underway but is about 80% finished now, we are close.
+- One of these mentorship projects involves a major change to the Cacti code base and architecture, namely the integration of the Cactus and Weaver modules into a common and comprehensive set of capabilities for cross-network operations. A first-cut version of these common COPM modules is [close to being merged](https://github.com/hyperledger-cacti/cacti/pull/3546) into the main branch.
```

Comment:
What does COPM stand for?

Reply:
@VRamakrishna @jenniferlianne

Reply:
COPM == Core OPerators Module. It's a package that represents intgrated functionality from Cactus and Weaver, and along the way should replace older legacy packages. It's being developed by a mentee (Jennifer Bell): https://lf-hyperledger.atlassian.net/wiki/spaces/INTERN/pages/21960881/Cacti+Core+Operators+Module+Project+Plan.